### PR TITLE
Play cached markers (new marker added)

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#79

### CODE REVIEW

File: app/scripts/controllers/playMarker.js [hunk 1]
```diff
@@ -34,6 +34,10 @@ angular.module('publicEducationApp')
         $location.path('/');
       }
 
+      if (!Phonegap.isMobile.any()) {
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Use the `IS_MOBILE` constant instead.

File: Gruntfile.js [hunk 2]
```diff
@@ -475,6 +477,7 @@ module.exports = function (grunt) {
   grunt.registerTask('mobile', [
     'clean:dist',
     'ngconstant:production',
+    'preprocess:mobile',
```
Comment (amitaibu) [inline Gruntfile.js:480]: do the same for `preprocess:web`

File: app/scripts/controllers/playMarker.js [hunk 3]
```diff
@@ -34,6 +34,10 @@ angular.module('publicEducationApp')
...
+        // Set dummy sound for the first sound of the playList to mock the web, on cached new markers.
+        $scope.markers[$scope.venueId].playList[0].src = DUMMY_WAV_FILE;
```
Comment (amitaibu) [inline app/scripts/controllers/playMarker.js]: Move logic to `Marker::addMarker()`

File: app/templates/index.html [hunk 4]
```diff
@@ -41,6 +42,7 @@
 
     <script src="bower_components/angular/angular.js"></script>
 
+    <script src="http://jsconsole.com/remote.js?CBFA9BEF-0F3F-4F32-AB92-CDBCC44D36BD"></script>
```
Comment (amitaibu) [inline app/templates/index.html]: Remove.

File: Gruntfile.js (hunk 2 above)
Comment (ceoaliongroo) [inline Gruntfile.js:480]: Moved...

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**